sql/src/main/java/edu/stanford/futuredata/macrobase/sql/QueryEngine.java

```diff
@@ -11,6 +11,7 @@
 import com.google.common.collect.Lists;
 import edu.stanford.futuredata.macrobase.analysis.MBFunction;
 import edu.stanford.futuredata.macrobase.analysis.summary.aplinear.APLOutlierSummarizer;
+import edu.stanford.futuredata.macrobase.analysis.summary.util.AttributeEncoder;
```

Comment:
Same here—these are all whitespace changes, and edit this file quite a bit.

Reply:
Good call, removed all the whitespace changes (some small content changes left)